airflow/www/views.py

```diff
@@ -2081,11 +2079,11 @@ class SlaMissModelView(wwwutils.SuperUserMixin, ModelViewOnly):
 @provide_session
 def _connection_ids(session=None):
     return [
-            (c.conn_id, c.conn_id)
-            for c in (
-                session.query(models.Connection.conn_id)
-                    .group_by(models.Connection.conn_id)
-            )
```

Comment:
These sort of white-space only changes make the PR much harder to review - in future it would be good to avoid "indent/layout only" changes.

Reply:
Agreed and I would be fine with that but the flake8 linting test that's built into the PR's kept failing without me making these changes. My guess is that these particular files haven't been touched in a while and so those tests never occurred.

Reply:
I though (perhaps wrongly) that flake8 was only applied to diffs, not the whole file. This is certainly what the contrib guides say anyway: `git diff upstream/master -u -- "*.py" | flake8 --diff`

Reply:
Hmmm. Good point, I'm not sure. This could be me be overzealous so that I could avoid issues in the builds after it failed a few times. I remember I stopped trying to use the flake8 test in the travis build after it failed a few times and started doing it locally so I could speed up the fixes. I most likely didn't attempt to use the diff flag. Point taken, I'll try to use the same commands next time.